ts-packages/web/src/app/(social)/_components/profile-section.tsx

```
@@ -37,15 +37,15 @@ export default function ProfileSection() {
+          <div className="w-20 h-20 rounded-full border border-neutral-500 bg-foreground-muted" />
         )}
       </div>
 
-      <div className="font-medium">{user.nickname}</div>
+      <div className="font-medium text-foreground">{user.nickname}</div>
 
       <div
         id="user-profile-description"
-        className="text-xs text-gray-400"
+        className="text-xs text-foreground-muted"
         dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)